store/src/java-test/com/zimbra/cs/mailbox/ContactTest.java

```diff
@@ -293,4 +303,22 @@ public void testEncodeContact() throws Exception {
         ToXML.encodeContact(response, new ItemIdFormatter(), new OperationContext(acct), contact, true, null);
         Assert.assertEquals(response.getElement("cn").getElement("a").getText(), "Cert1149638887753217");
     }
+
+    @Test
+    public void testTruncatedContactsTgzImport() throws IOException {
+        File file = new File("src/java-test/Truncated.tgz");
```

Comment:
Truncated.tgz is missing; need to commit the file as well.

Reply:
Done